File: ChatApp.Server/Services/ViewModels/Groups/GroupViewModel.cs
using ChatApp.Server.Services.ViewModels.Users;

namespace ChatApp.Server.Services.ViewModels.Groups
{
    public class GroupViewModel
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public UserViewModel? CreatedByUser { get; set; }

        //public Media? ProfilePicture { get; set; }

        public List<int> GroupUsersId { get; set; } = new List<int>();

        public DateTime CreatedAt { get; set; }

        public DateTime ModifiedAt { get; set; }
    }
}

File: chatapp.client/src/app/models/group-view-model.ts
import { UserViewModel } from "./user-view-model";

export class GroupViewModel {
  id: number = 0;
  name: string = '';
  createdByUser: UserViewModel | null = null;
  groupUsersId: number[] = [];
  createdAt: string = '';
  modifiedAt: string = '';
}
